src/data/balance/signature-species.ts

```diff
@@ -11,87 +11,87 @@ export type SignatureSpecies = {
-  VOLKNER: [Species.SHINX, Species.CHINCHOU, Species.ROTOM],
+  FANTINA: [Species.MISDREAVUS, Species.DRIFLOON, Species.DUSKULL, Species.SPIRITOMB],
+  BYRON: [Species.SHIELDON, Species.BRONZOR, Species.SKARMORY, Species.ARON],
+  CANDICE: [Species.SNORUNT, Species.SNOVER, Species.SNEASEL, Species.GLACEON],
+  VOLKNER: [Species.ELEKID, Species.SHINX, Species.JOLTEON, Species.ROTOM],
   // Gym Leaders- Unova
-  CILAN: [Species.PANSAGE, Species.FOONGUS, Species.PETILIL],
-  CHILI: [Species.PANSEAR, Species.DARUMAKA, Species.NUMEL],
-  CRESS: [Species.PANPOUR, Species.TYMPOLE, Species.SLOWPOKE],
-  CHEREN: [Species.LILLIPUP, Species.MINCCINO, Species.PIDOVE],
-  LENORA: [Species.PATRAT, Species.DEERLING, Species.AUDINO],
-  ROXIE: [Species.VENIPEDE, Species.TRUBBISH, Species.SKORUPI],
-  BURGH: [Species.SEWADDLE, Species.SHELMET, Species.KARRABLAST],
-  ELESA: [Species.EMOLGA, Species.BLITZLE, Species.JOLTIK],
-  CLAY: [Species.DRILBUR, Species.SANDILE, Species.GOLETT],
-  SKYLA: [Species.DUCKLETT, Species.WOOBAT, Species.RUFFLET],
-  BRYCEN: [Species.CRYOGONAL, Species.VANILLITE, Species.CUBCHOO],
-  DRAYDEN: [Species.DRUDDIGON, Species.AXEW, Species.DEINO],
-  MARLON: [Species.WAILMER, Species.FRILLISH, Species.TIRTOUGA],
+  CILAN: [Species.PANSAGE, Species.SNIVY, Species.MARACTUS, Species.FERROSEED],
+  CHILI: [Species.PANSEAR, Species.TEPIG, Species.HEATMOR, Species.DARUMAKA],
+  CRESS: [Species.PANPOUR, Species.OSHAWOTT, Species.BASCULIN, Species.TYMPOLE],
+  CHEREN: [Species.LILLIPUP, Species.PIDOVE, Species.MINCCINO, Species.BOUFFALANT],
+  LENORA: [Species.PATRAT, Species.DEERLING, Species.AUDINO, Species.BRAVIARY],
+  ROXIE: [Species.VENIPEDE, Species.KOFFING, Species.TRUBBISH, Species.TOXEL],
+  BURGH: [Species.SEWADDLE, Species.DWEBBLE, [Species.KARRABLAST, Species.SHELMET], Species.DURANT],
+  ELESA: [Species.BLITZLE, Species.EMOLGA, Species.JOLTIK, Species.TYNAMO],
+  CLAY: [Species.DRILBUR, Species.SANDILE, Species.TYMPOLE, Species.GOLETT],
+  SKYLA: [Species.DUCKLETT, Species.PIDOVE, Species.WOOBAT, [Species.RUFFLET, Species.VULLABY]],
+  BRYCEN: [Species.CRYOGONAL, Species.VANILLITE, Species.CUBCHOO, Species.GALAR_DARUMAKA],
+  DRAYDEN: [Species.AXEW, Species.DRUDDIGON, Species.TRAPINCH, Species.DEINO],
+  MARLON: [Species.FRILLISH, Species.TIRTOUGA, Species.WAILMER, Species.MANTYKE],
   // Gym Leaders- Kalos
-  VIOLA: [Species.SURSKIT, Species.SCATTERBUG],
-  GRANT: [Species.AMAURA, Species.TYRUNT],
-  KORRINA: [Species.HAWLUCHA, Species.LUCARIO, Species.MIENFOO],
-  RAMOS: [Species.SKIDDO, Species.HOPPIP, Species.BELLSPROUT],
-  CLEMONT: [Species.HELIOPTILE, Species.MAGNEMITE, Species.EMOLGA],
-  VALERIE: [Species.SYLVEON, Species.MAWILE, Species.MR_MIME],
-  OLYMPIA: [Species.ESPURR, Species.SIGILYPH, Species.SLOWKING],
-  WULFRIC: [Species.BERGMITE, Species.SNOVER, Species.CRYOGONAL],
+  VIOLA: [Species.SCATTERBUG, Species.SURSKIT, Species.CUTIEFLY, Species.DWEBBLE],
```

Comment:
```suggestion
  VIOLA: [Species.SCATTERBUG, Species.SURSKIT, Species.CUTIEFLY, Species.BLIPBUG],
```
No one else using that mon, kinda fits 🤷‍♂️